modules/apps/account/account-test/src/testFunctional/tests/2BAccount.testcase

```diff
@@ -722,6 +722,70 @@ definition {
 		}
 	}
 
+	@description = "This test asserts an account member can search accounts they belong to"
+	@priority = "3"
+	test AccountMemberCanSearchForAnAccount {
+		task ("Given a new account is created and a new account user is assigned to two accounts") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "test@liferay.com",
+				userLoginFullName = "Test Test");
+
+			JSONAccountEntry.addAccountEntry(
+				accountEntryName = "Account Test",
+				accountEntryType = "Business");
+
+			JSONAccountEntryUser.addAccountEntryUser(
+				accountEntryName = "Account Test",
+				userEmailAddress = "user1@liferay.com",
+				userFirstName = "user1",
+				userLastName = "user1",
+				userScreenName = "user1");
+
+			Navigator.gotoPage(pageName = "2B Account Page");
+
+			LexiconEntry.gotoEntry(rowEntry = "Account Name");
+
+			Accounts.assignUsers(
+				accountName = "Account Name",
+				assigneeName = "user1");
+		}
+
+		task ("When the new account member signs in and views the accounts they are a part of through the Account Management widget") {
+			JSONUser.setFirstPassword(
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "user1@liferay.com");
```

Comment:
Move ahead into the Given task block, thanks.
Should the _property custom.properties = "jsonws.web.service.paths.excludes=";_ required for this macro?